utils/api.py

```diff
@@ -237,29 +232,12 @@ def reload_if_needed(self):
         if self._data is None:
             self.reload()
 
+    def filter_by_query_string(self, **params):
+        """Specify query string to use with collection."""
+        return SearchResult(self, self._api.get(self._href, **params))
+
     def find_by(self, **params):
         """Search items in collection. Filters based on keywords passed."""
-        if self._api.version == "2.0.0-pre":
-            # Special case, there can be both, so try sqlfilter first and if that does not work ...
-            try:
-                return self._find_by_sqlfilter(**params)
-            except APIException:
-                return self._find_by_filter(**params)
-        elif self._api.version.is_in_series("1.1") or self._api.version >= "2.0.0":
-            # New function
-            return self._find_by_filter(**params)
-        else:
-            # Old function
-            return self._find_by_sqlfilter(**params)
-
-    def _find_by_sqlfilter(self, **params):
```

Comment:
`_find_by_sqlfilter` is no longer needed, it was valid for old versions of api that is no longer used by any version of CFME that we test. Also the logic for choosing appropriate method can be removed and what used to be `_find_by_filter` internal method can now be directly `find_by`. `_find_by_sqlfilter` and `_find_by_filter` internal methods are not used directly anywhere, always through `find_by`, so it's safe to remove them.